python/test/conftest.py

```diff
@@ -14,3 +16,27 @@ def test_environment():
     worksapce = os.environ.get("TIMEPLUS_WORKSPACE")
 
     return Environment().address(api_address).apikey(api_key).workspace(worksapce)
+
+
+@pytest.fixture
+def test_table(test_environment):
+    # Create a new stream
+    stream_name = "test_stream"
+    stream = (
+        Stream(env=test_environment)
+        .name(stream_name)
+        .column("id", "integer")
+        .column("data", "string")
+        .create()
+    )
+
+    # Generate and ingest some random data
+    data = ["id", "data"]
+    values = [[i, f"random_string_{random.randint(0, 1000)}"] for i in range(10)]
```

Comment:
you dont have to generate random value for test data, you can just use some fixed value which can be easily verify in your query